src/ng/filter/filters.js

```diff
@@ -358,10 +358,11 @@ function dateGetter(name, size, offset, trim) {
   };
 }
 
-function dateStrGetter(name, shortForm) {
+function dateStrGetter(name, shortForm, formatProp) {
   return function(date, formats) {
     var value = date['get' + name]();
-    var get = uppercase(shortForm ? ('SHORT' + name) : name);
+    var propName = formatProp || name;
+    var get = uppercase(shortForm ? ('SHORT' + propName) : propName);
```

Comment:
If we change this to
```js
function dateStrGetter(name, shortForm, propPrefix) {
...
var get = uppercase((propPrefix || '') + (shortForm ? 'Short' : '') + name);
...
```

And also change the line below to
```js
LLLL: dateStrGetter('Month', false, 'Standalone'),
```

Then, in the future, we can implement *LLL* by just doing
```js
LLL: dateStrGetter('Month', true, 'Standalone'),
```

[To change *Standalone* to a boolean is also fine]